internal/sbom/spdx.go

```diff
@@ -40,82 +38,189 @@ func GenerateSPDX(koVersion string, date time.Time, mod []byte, imgDigest v1.Has
 		return nil, err
 	}
 
+	mainPackageID := "SPDXRef-Package-" + strings.ReplaceAll(bi.Main.Path, "/", ".")
+
+	doc := Document{
+		Version:           "SPDX-2.2",
+		DataLicense:       "CC0-1.0",
+		ID:                "SPDXRef-DOCUMENT",
+		Name:              bi.Main.Path,
+		Namespace:         "http://spdx.org/spdxdocs/" + bi.Main.Path,
+		DocumentDescribes: []string{mainPackageID},
+		CreationInfo: CreationInfo{
+			Created:  date.Format(dateFormat),
+			Creators: []string{"Tool: ko " + koVersion},
+		},
+		Packages:      make([]Package, 0, 1+len(bi.Deps)),
+		Relationships: make([]Relationship, 0, 1+len(bi.Deps)),
+	}
+
+	doc.Relationships = append(doc.Relationships, Relationship{
+		Element: "SPDXRef-DOCUMENT",
+		Type:    "DESCRIBES",
+		Related: mainPackageID,
+	})
+
+	doc.Packages = append(doc.Packages, Package{
+		Name: bi.Main.Path,
+		ID:   mainPackageID,
+		// TODO: PackageSupplier: "Organization: " + bs.Main.Path
+		DownloadLocation: "https://" + bi.Main.Path,
+		FilesAnalyzed:    false,
+		// TODO: PackageHomePage:  "https://" + bi.Main.Path,
+		LicenseConcluded: "NOASSERTION",
+		LicenseDeclared:  "NOASSERTION",
+		// TODO: PackageLicenseComments: "NOASSERTION"
```

Comment:
Why these TODOs?

Reply:
We had these fields in what we emitted before, and I didn't see corresponding fields in the JSON types (cc @puerco )

I can drop them since NOASSERTION is not more helpful 🤣

Reply:
I left the ones that weren't NOASSERTION and removed the rest